content/sensu-go/6.2/observability-pipeline/observe-events/events.md

```diff
@@ -1896,7 +1900,8 @@ total_state_change: 0
 
 executed     |      |
 -------------|------
-description  | Time at which the check request was executed. In seconds since the Unix epoch.{{% notice note %}}**NOTE**: For events created with the [events API](../../../api/events/), the `executed` value is `0`.
+description  | Time at which the check request was executed. In seconds since the Unix epoch.{{% notice note %}}
+**NOTE**: For events created with the [events API](../../../api/events/), the `executed` value is `0`.
```

Comment:
I don't think this is true. If I provide an `executed` value for the check in the event definition, that value will be included in history.executed.